src/accountsdb/buffer_pool.zig

```diff
@@ -19,26 +21,75 @@ comptime {
     _ = offset;
 }
 
+const FrameRef = packed struct(u32) {
+    f_idx: FrameIndex,
+    hit: bool,
+    const INIT = FrameRef{
+        .f_idx = INVALID_FRAME,
+        .hit = false,
+    };
+};
+
+const Rc = std.atomic.Value(u32);
+
 const LinuxIoMode = enum {
     Blocking,
     IoUring,
 };
-const linux_io_mode: LinuxIoMode = .IoUring;
+const linux_io_mode: LinuxIoMode = .Blocking;
 
+// TODO: ideally we should be able to select this with a cli flag. (#509)
 const use_io_uring = builtin.os.tag == .linux and linux_io_mode == .IoUring;
 
+const io_uring_entries = 128;
```

Comment:
screaming case form constants?

Reply:
made them screaming case f40689214b7f8a0945d24c5a2847c29af6db0203

Reply:
missed this last review, but can we keep all constants at the top of the file

Reply:
55f52cf76c01ca23a88562a668737b36f28ca235